src/arch/arm/cortexa8/mmu.c

```diff
@@ -48,18 +45,6 @@ static int mmu_init(void) {
 	);
 
 	return 0;
-
-	/* Setup physical address of the first level translation table */
-	__asm__ __volatile__ (
```

Comment:
There's setup of P15 register.
The physical address of the translation table is unique for each task and it's set on `mmu_set_context()`.